测速日志 2025-10-29 09:47:02
[2025-10-29 09:47:05] ✅ 测速成功: http://61.221.215.25:8800/hls/43/index.m3u8
   速度: 236.33KB/s | 数据量: 0.2KB | 总耗时: 0.34s
[2025-10-29 09:47:05] ✅ 测速成功: http://8.138.7.223/【公众号：榴芒电视】/cctv1.m3u8
   速度: 108.40KB/s | 数据量: 0.1KB | 总耗时: 0.43s
[2025-10-29 09:47:05] ✅ 测速成功: http://8.138.7.223/tv/cctv7.m3u8
   速度: 132.81KB/s | 数据量: 0.1KB | 总耗时: 0.43s
[2025-10-29 09:47:05] ✅ 测速成功: http://61.162.52.6:8088/newlive/live/hls/60/live.m3u8
   速度: 232.42KB/s | 数据量: 0.2KB | 总耗时: 0.45s
[2025-10-29 09:47:05] ✅ 测速成功: http://123.146.162.24:8017/c2F0hmi/1000/live.m3u8?zzhongqd
   速度: 438.48KB/s | 数据量: 0.4KB | 总耗时: 0.45s
[2025-10-29 09:47:05] ✅ 测速成功: http://8.138.7.223/【公众号：榴芒电视】/cctv7.m3u8
   速度: 209.96KB/s | 数据量: 0.2KB | 总耗时: 0.46s
[2025-10-29 09:47:05] ✅ 测速成功: http://123.146.162.24:8013/tslslive/PU2vzMI/hls/live_sd.m3u8
   速度: 545.90KB/s | 数据量: 0.5KB | 总耗时: 0.50s
[2025-10-29 09:47:05] ✅ 测速成功: http://123.146.162.24:8017/c2F0hmi/1000/live.m3u8
   速度: 438.48KB/s | 数据量: 0.4KB | 总耗时: 0.50s
[2025-10-29 09:47:06] ✅ 测速成功: http://123.146.162.24:8013/tslslive/PU2vzMI/hls/live_sd.m3u8?zzhongqd
   速度: 545.90KB/s | 数据量: 0.5KB | 总耗时: 0.50s
[2025-10-29 09:47:06] ✅ 测速成功: http://36.32.174.67:60080/newlive/live/hls/33/live.m3u8
   速度: 232.42KB/s | 数据量: 0.2KB | 总耗时: 0.69s
[2025-10-29 09:47:06] ✅ 测速成功: http://112.27.235.94:8000/hls/25/index.m3u8
   速度: 240.23KB/s | 数据量: 0.2KB | 总耗时: 0.36s
[2025-10-29 09:47:06] ✅ 测速成功: http://112.27.235.94:8000/hls/18/index.m3u8
   速度: 240.23KB/s | 数据量: 0.2KB | 总耗时: 0.40s
[2025-10-29 09:47:06] ✅ 测速成功: http://8.138.7.223/【公众号：榴芒电视】/农林卫视.m3u8
   速度: 118.16KB/s | 数据量: 0.1KB | 总耗时: 0.46s
[2025-10-29 09:47:06] ✅ 测速成功: http://123.146.162.24:8013/tslslive/noEX9SG/hls/live_sd.m3u8?zzhongqd
   速度: 545.90KB/s | 数据量: 0.5KB | 总耗时: 0.46s
[2025-10-29 09:47:06] ✅ 测速成功: http://183.188.111.173:9003/hls/46/index.m3u8
   速度: 183.59KB/s | 数据量: 0.2KB | 总耗时: 0.46s
[2025-10-29 09:47:06] ✅ 测速成功: http://60.16.0.216:5757/hls/39/index.m3u8
   速度: 240.23KB/s | 数据量: 0.2KB | 总耗时: 0.47s
[2025-10-29 09:47:06] ✅ 测速成功: http://36.32.174.67:60080/newlive/live/hls/24/live.m3u8
   速度: 232.42KB/s | 数据量: 0.2KB | 总耗时: 0.73s
[2025-10-29 09:47:06] ✅ 测速成功: http://8.138.7.223/tv/gzjs.m3u8
   速度: 133.79KB/s | 数据量: 0.1KB | 总耗时: 0.43s
[2025-10-29 09:47:06] ✅ 测速成功: http://36.32.174.67:60080/newlive/live/hls/45/live.m3u8
   速度: 232.42KB/s | 数据量: 0.2KB | 总耗时: 0.76s
[2025-10-29 09:47:06] ✅ 测速成功: http://36.32.174.67:60080/newlive/live/hls/49/live.m3u8
   速度: 232.42KB/s | 数据量: 0.2KB | 总耗时: 0.77s
[2025-10-29 09:47:06] ✅ 测速成功: http://111.202.12.159:8080/kYGHWzA/1000/live.m3u8
   速度: 331.05KB/s | 数据量: 0.3KB | 总耗时: 0.44s
[2025-10-29 09:47:06] ✅ 测速成功: http://111.202.12.159:8080/3zvkK28/1000/live.m3u8
   速度: 69.34KB/s | 数据量: 0.1KB | 总耗时: 0.49s
[2025-10-29 09:47:07] ✅ 测速成功: http://36.32.174.67:60080/newlive/live/hls/38/live.m3u8
   速度: 232.42KB/s | 数据量: 0.2KB | 总耗时: 0.75s
[2025-10-29 09:47:07] ✅ 测速成功: http://8.138.7.223/tv/hxws.m3u8
   速度: 211.91KB/s | 数据量: 0.2KB | 总耗时: 0.43s
[2025-10-29 09:47:07] ✅ 测速成功: http://36.32.174.67:60080/newlive/live/hls/47/live.m3u8
   速度: 232.42KB/s | 数据量: 0.2KB | 总耗时: 0.77s
[2025-10-29 09:47:07] ✅ 测速成功: http://36.32.174.67:60080/newlive/live/hls/22/live.m3u8
   速度: 232.42KB/s | 数据量: 0.2KB | 总耗时: 0.76s
[2025-10-29 09:47:07] ✅ 测速成功: http://111.53.96.67:8081/live/1/index.m3u8
   速度: 208.98KB/s | 数据量: 0.2KB | 总耗时: 0.41s
[2025-10-29 09:47:07] ✅ 测速成功: http://36.32.174.67:60080/newlive/live/hls/36/live.m3u8
   速度: 232.42KB/s | 数据量: 0.2KB | 总耗时: 0.81s
[2025-10-29 09:47:07] ✅ 测速成功: http://36.32.174.67:60080/newlive/live/hls/46/live.m3u8
   速度: 232.42KB/s | 数据量: 0.2KB | 总耗时: 0.78s
[2025-10-29 09:47:07] ✅ 测速成功: http://111.53.96.67:8081/live/1/index.m3u8?%E5%85%B3%E6%B3%A8%E5%BE%AE%E4%BF%A1%E5%85%AC%E4%BC%97%E5%8F%B7[%E6%99%B4%E5%9B%AD]
   速度: 208.98KB/s | 数据量: 0.2KB | 总耗时: 0.41s
[2025-10-29 09:47:07] ✅ 测速成功: http://36.32.174.67:60080/newlive/live/hls/48/live.m3u8
   速度: 232.42KB/s | 数据量: 0.2KB | 总耗时: 0.79s
[2025-10-29 09:47:07] ✅ 测速成功: http://36.32.174.67:60080/newlive/live/hls/34/live.m3u8
   速度: 232.42KB/s | 数据量: 0.2KB | 总耗时: 0.79s
[2025-10-29 09:47:10] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.138:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.138:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c741a50>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:10] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.107:8000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.107:8000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c741150>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:10] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.108:8000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.108:8000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c740d50>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:10] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.117:8000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.117:8000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c741d50>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:10] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.118:8000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.118:8000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c741b50>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:10] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.119:8000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.119:8000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c741750>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:10] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.120:8000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.120:8000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c740c50>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:10] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.122:8000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.122:8000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c741250>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:10] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.1.23:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.1.23:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c740f50>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:12] ✅ 测速成功: http://122.53.222.9:9981/stream/channelid/2084767267?profile=pass
   速度: 544.43KB/s | 数据量: 3072.0KB | 总耗时: 6.08s
[2025-10-29 09:47:13] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.1.29:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.1.29:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c741550>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:13] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.129:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.129:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c741e50>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:13] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.130:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.130:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c742250>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:13] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.1.3:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.1.3:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c742050>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:13] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.131:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.131:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c741450>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:13] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.1.5:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.1.5:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c742550>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:13] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.18:9610
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.18:9610 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c742950>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:13] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.1.6:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.1.6:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c742d50>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:13] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.1.254:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.1.254:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c743150>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:15] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.1.8:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.1.8:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c740950>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:16] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.53:9814
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.53:9814 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c743450>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:16] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.134:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.134:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c743750>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:16] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.1.11:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.1.11:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c743a50>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:16] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.135:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.135:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c743d50>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:16] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.1.13:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.1.13:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61c150>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:16] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.136:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.136:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61c450>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:16] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.1.15:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.1.15:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61c750>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:16] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.132:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.132:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61ca50>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:16] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.139:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.139:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61cd50>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:18] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.140:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.140:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c742d50>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:19] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.141:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.141:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c741450>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:19] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.142:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.142:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c743c50>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:19] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.143:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.143:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c743950>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:19] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.145:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.145:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c743650>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:19] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.146:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.146:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c743350>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:19] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.147:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.147:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c742f50>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:19] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.148:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.148:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c742b50>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:19] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.188:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.188:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c742750>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:19] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.204:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.204:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c741950>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:21] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.185:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.185:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c741150>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:22] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.149:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.149:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61cb50>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:22] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.144:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.144:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61c650>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:22] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.1.49:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.1.49:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61c250>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:22] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.1.53:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.1.53:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61d150>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:22] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.1.35:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.1.35:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61d450>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:22] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.1.40:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.1.40:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c743350>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:22] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.1.41:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.1.41:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c743c50>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:22] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.1.43:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.1.43:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c742d50>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:22] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.1.44:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.1.44:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c741750>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:24] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.1.45:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.1.45:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c742a50>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:25] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.1.51:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.1.51:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c740f50>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:25] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.1.52:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.1.52:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c743550>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:25] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.1.235:6000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.1.235:6000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c743850>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:25] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.0.25:8192
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.0.25:8192 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c743b50>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:25] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.0.183:9434
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.0.183:9434 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c743e50>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:25] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.0.26:8200
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.0.26:8200 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c742950>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:25] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.0.9:8064
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.0.9:8064 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61d050>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:25] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.19:9614
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.19:9614 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61c550>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:25] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.50:9802
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.50:9802 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61cc50>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:27] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.54:9818
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.54:9818 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61c750>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:28] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.55:9822
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.55:9822 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61d750>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:28] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.56:9826
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.56:9826 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c740f50>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:28] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.52:9810
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.52:9810 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c742d50>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:28] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.16:9602
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.16:9602 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c742550>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:28] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.12:9418
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.12:9418 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c743750>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:28] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.20:9618
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.20:9618 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c741e50>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:28] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.11:9414
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.11:9414 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c743250>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:28] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.17:9606
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.17:9606 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c741150>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:28] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.13:9422
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.13:9422 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c741950>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:29] ✅ 测速成功: http://221.13.235.158:9901/tsfile/live/0002_1.m3u8?key=txiptv&playlive=1&authid=0
   速度: 408.20KB/s | 数据量: 0.4KB | 总耗时: 0.49s
[2025-10-29 09:47:29] ✅ 测速成功: http://221.13.235.158:9901/tsfile/live/0003_1.m3u8?key=txiptv&playlive=1&authid=0
   速度: 408.20KB/s | 数据量: 0.4KB | 总耗时: 0.45s
[2025-10-29 09:47:30] ✅ 测速成功: http://221.13.235.158:9901/tsfile/live/0004_1.m3u8?key=txiptv&playlive=1&authid=0
   速度: 408.20KB/s | 数据量: 0.4KB | 总耗时: 0.45s
[2025-10-29 09:47:30] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.7:9306
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.7:9306 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c742750>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:31] ✅ 测速成功: http://221.13.235.158:9901/tsfile/live/0005_1.m3u8?key=txiptv&playlive=1&authid=0
   速度: 408.20KB/s | 数据量: 0.4KB | 总耗时: 0.46s
[2025-10-29 09:47:31] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.15:9430
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.15:9430 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c742b50>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:31] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.57:9830
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.57:9830 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61c450>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:31] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.48:8000
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.48:8000 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61c950>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:31] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.14:9426
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.14:9426 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61c050>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:31] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.8:9632
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.8:9632 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c743650>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:31] ❌ 测速失败: http://49.77.104.8:8554/rtp/239.49.8.35:8620
   错误: HTTPConnectionPool(host='49.77.104.8', port=8554): Max retries exceeded with url: /rtp/239.49.8.35:8620 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61c250>, 'Connection to 49.77.104.8 timed out. (connect timeout=3.05)')) | 域名: 49.77.104.8
[2025-10-29 09:47:31] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/0002_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/0002_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61da50>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:32] ✅ 测速成功: http://221.13.235.158:9901/tsfile/live/0006_1.m3u8?key=txiptv&playlive=1&authid=0
   速度: 408.20KB/s | 数据量: 0.4KB | 总耗时: 0.45s
[2025-10-29 09:47:32] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/0003_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/0003_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c742550>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:32] ✅ 测速成功: http://221.13.235.158:9901/tsfile/live/0008_1.m3u8?key=txiptv&playlive=1&authid=0
   速度: 408.20KB/s | 数据量: 0.4KB | 总耗时: 0.45s
[2025-10-29 09:47:32] ✅ 测速成功: http://221.13.235.158:9901/tsfile/live/1005_1.m3u8?key=txiptv&playlive=1&authid=0
   速度: 422.85KB/s | 数据量: 0.4KB | 总耗时: 0.49s
[2025-10-29 09:47:32] ✅ 测速成功: http://221.13.235.158:9901/tsfile/live/0010_1.m3u8?key=txiptv&playlive=1&authid=0
   速度: 413.09KB/s | 数据量: 0.4KB | 总耗时: 0.50s
[2025-10-29 09:47:32] ✅ 测速成功: http://221.13.235.158:9901/tsfile/live/0011_1.m3u8?key=txiptv&playlive=1&authid=0
   速度: 413.09KB/s | 数据量: 0.4KB | 总耗时: 0.48s
[2025-10-29 09:47:33] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/0004_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/0004_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c743250>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:33] ✅ 测速成功: http://221.13.235.158:9901/tsfile/live/0012_1.m3u8?key=txiptv&playlive=1&authid=0
   速度: 413.09KB/s | 数据量: 0.4KB | 总耗时: 0.46s
[2025-10-29 09:47:33] ✅ 测速成功: http://221.13.235.158:9901/tsfile/live/0013_1.m3u8?key=txiptv&playlive=1&authid=0
   速度: 413.09KB/s | 数据量: 0.4KB | 总耗时: 0.45s
[2025-10-29 09:47:34] ✅ 测速成功: http://221.13.235.158:9901/tsfile/live/0014_1.m3u8?key=txiptv&playlive=1&authid=0
   速度: 413.09KB/s | 数据量: 0.4KB | 总耗时: 0.49s
[2025-10-29 09:47:34] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/0005_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/0005_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c743950>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:34] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/0006_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/0006_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c740c50>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:34] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/0007_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/0007_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c741550>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:34] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/0008_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/0008_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c742050>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:34] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/0009_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/0009_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c743350>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:35] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/0010_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/0010_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61cb50>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:35] ✅ 测速成功: http://221.13.235.158:9901/tsfile/live/0015_1.m3u8?key=txiptv&playlive=1&authid=0
   速度: 413.09KB/s | 数据量: 0.4KB | 总耗时: 0.49s
[2025-10-29 09:47:35] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/0011_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/0011_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c743a50>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:35] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/0012_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/0012_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c741b50>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:36] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/0013_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/0013_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c741750>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:37] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/0014_2.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/0014_2.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61d450>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:37] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/0015_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/0015_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61c350>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:37] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/0016_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/0016_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61c750>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:37] ✅ 测速成功: http://221.13.235.158:9901/tsfile/live/0107_1.m3u8?key=txiptv&playlive=1&authid=0
   速度: 417.97KB/s | 数据量: 0.4KB | 总耗时: 0.45s
[2025-10-29 09:47:37] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/1041_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/1041_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61d050>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:37] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/1008_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/1008_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c742050>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:38] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/1006_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/1006_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c743950>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:38] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/1040_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/1040_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c742550>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:38] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/1042_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/1042_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c743850>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:38] ✅ 测速成功: http://221.13.235.158:9901/tsfile/live/0122_1.m3u8?key=txiptv&playlive=1&authid=0
   速度: 417.97KB/s | 数据量: 0.4KB | 总耗时: 0.45s
[2025-10-29 09:47:38] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/1001_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/1001_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c743450>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:39] ✅ 测速成功: http://221.13.235.158:9901/tsfile/live/0130_1.m3u8?key=txiptv&playlive=1&authid=0
   速度: 417.97KB/s | 数据量: 0.4KB | 总耗时: 0.49s
[2025-10-29 09:47:39] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/0137_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/0137_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c742250>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:40] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/0107_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/0107_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c742e50>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:40] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/0119_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/0119_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c741f50>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:40] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/0115_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/0115_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c741a50>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:40] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/0122_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/0122_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c740f50>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:41] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/0129_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/0129_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61cc50>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:41] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/1002_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/1002_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61cd50>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:41] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/0112_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/0112_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61d250>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:41] ✅ 测速成功: http://221.13.235.158:9901/tsfile/live/0127_1.m3u8?key=txiptv&playlive=1&authid=0
   速度: 417.97KB/s | 数据量: 0.4KB | 总耗时: 0.46s
[2025-10-29 09:47:41] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/0130_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/0130_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c741e50>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:41] ✅ 测速成功: http://221.13.235.158:9901/tsfile/live/0139_1.m3u8?key=txiptv&playlive=1&authid=0
   速度: 417.97KB/s | 数据量: 0.4KB | 总耗时: 0.44s
[2025-10-29 09:47:41] ✅ 测速成功: http://221.13.235.158:9901/tsfile/live/0124_1.m3u8?key=txiptv&playlive=1&authid=0
   速度: 417.97KB/s | 数据量: 0.4KB | 总耗时: 0.46s
[2025-10-29 09:47:42] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/0118_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/0118_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61dd50>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:42] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/0017_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/0017_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61c050>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:43] ✅ 测速成功: http://221.13.235.158:9901/tsfile/live/0128_1.m3u8?key=txiptv&playlive=1&authid=0
   速度: 417.97KB/s | 数据量: 0.4KB | 总耗时: 0.50s
[2025-10-29 09:47:43] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/0113_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/0113_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61e050>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:43] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/0110_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/0110_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c742e50>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:43] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/0127_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/0127_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c743850>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:44] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/0138_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/0138_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c742750>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:44] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/0139_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/0139_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c743d50>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:44] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/0124_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/0124_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c743650>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:44] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/1000_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/1000_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c742b50>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:44] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/0126_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/0126_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c742050>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:45] ✅ 测速成功: http://221.13.235.158:9901/tsfile/live/1014_1.m3u8?key=txiptv&playlive=1&authid=0
   速度: 422.85KB/s | 数据量: 0.4KB | 总耗时: 0.49s
[2025-10-29 09:47:45] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/0128_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/0128_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c743c50>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:45] ✅ 测速成功: http://221.13.235.158:9901/tsfile/live/1012_1.m3u8?key=txiptv&playlive=1&authid=0
   速度: 422.85KB/s | 数据量: 0.4KB | 总耗时: 0.50s
[2025-10-29 09:47:45] ✅ 测速成功: http://221.13.235.158:9901/tsfile/live/1008_1.m3u8?key=txiptv&playlive=1&authid=0
   速度: 422.85KB/s | 数据量: 0.4KB | 总耗时: 0.45s
[2025-10-29 09:47:45] ✅ 测速成功: http://221.13.235.158:9901/tsfile/live/1009_1.m3u8?key=txiptv&playlive=1&authid=0
   速度: 422.85KB/s | 数据量: 0.4KB | 总耗时: 0.46s
[2025-10-29 09:47:45] ✅ 测速成功: http://221.13.235.158:9901/tsfile/live/1010_1.m3u8?key=txiptv&playlive=1&authid=0
   速度: 422.85KB/s | 数据量: 0.4KB | 总耗时: 0.49s
[2025-10-29 09:47:45] ✅ 测速成功: http://221.13.235.158:9901/tsfile/live/1007_1.m3u8?key=txiptv&playlive=1&authid=0
   速度: 422.85KB/s | 数据量: 0.4KB | 总耗时: 0.49s
[2025-10-29 09:47:46] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/0141_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/0141_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c742a50>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:46] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/0111_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/0111_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c741550>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:46] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/0121_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/0121_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61da50>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:47] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/0136_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/0136_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61c650>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:47] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/0140_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/0140_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61c150>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:47] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/0143_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/0143_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61ca50>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:47] ✅ 测速成功: http://221.13.235.158:9901/tsfile/live/1017_1.m3u8?key=txiptv&playlive=1&authid=0
   速度: 422.85KB/s | 数据量: 0.4KB | 总耗时: 0.45s
[2025-10-29 09:47:48] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/1005_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/1005_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c742b50>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:48] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/1007_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/1007_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c743d50>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:48] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/1010_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/1010_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c743850>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:48] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/1009_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/1009_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c743250>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:49] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/1003_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/1003_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c742950>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:49] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/1013_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/1013_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c741b50>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:49] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/1014_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/1014_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c741750>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:50] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/1028_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/1028_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c741450>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:50] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/1020_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/1020_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c741a50>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:50] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/1037_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/1037_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c743750>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:51] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/1030_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/1030_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61e350>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:51] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/faacts/1035_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/faacts/1035_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61d450>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:51] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/1032_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/1032_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61c550>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:51] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/1033_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/1033_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61d350>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:52] ✅ 测速成功: http://221.13.235.158:9901/tsfile/live/1016_1.m3u8?key=txiptv&playlive=1&authid=0
   速度: 422.85KB/s | 数据量: 0.4KB | 总耗时: 0.50s
[2025-10-29 09:47:52] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/1034_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/1034_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61c950>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:52] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/1038_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/1038_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61c050>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:53] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/1012_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/1012_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c742550>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:53] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/1036_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/1036_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c61e650>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:53] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/1017_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/1017_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c741450>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:53] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/1019_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/1019_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c742950>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:54] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/1025_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/1025_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c742250>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
[2025-10-29 09:47:54] ❌ 测速失败: http://119.125.134.179:9901/tsfile/live/1021_1.m3u8?key=txiptv&playlive=1&authid=0
   错误: HTTPConnectionPool(host='119.125.134.179', port=9901): Max retries exceeded with url: /tsfile/live/1021_1.m3u8?key=txiptv&playlive=1&authid=0 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f718c741950>, 'Connection to 119.125.134.179 timed out. (connect timeout=3.05)')) | 域名: 119.125.134.179
